src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -60,6 +60,32 @@ public class C2
             End Using
         End Function
 
+        <WpfTheory, CombinatorialData>
+        <Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function CompletionOnListPattern_FirstNested(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document>
+public class C
+{
+}
+public class C2
+{
+    public C2 CProperty { get; set; }
+    public int IntProperty { get; set; }
+    void M(C c)
+    {
+        _ = c is { $$
+    }
+}
+                </Document>,
+                showCompletionInArgumentLists:=showCompletionInArgumentLists, languageVersion:=LanguageVersion.Preview)
+
+                ' We don't yet offer expression completion in this context (could be a constant for a list-pattern)
+                state.SendTypeChars("CP")
```

Comment:
seems... bad :)  it's not blocking, but i would say this is a basic completion scenario we want to get working by ship time.

Reply:
Why would this work in this branch or `main` branch (ie. without support for list-patterns)?

Reply:
sorry, the comment int eh code confused me.  it sounded like you were saying this shoudln't work even with list patterns.

I would instead have this test state that this is expected behavior *prior* to list pattenrs comining online.  Then we shoudl have a tracking bug that ensures that this scenario works when we do list patterns :)

Reply:
Sorry the comment wasn't clear. I'll tweak the wording.

There's no need for tracking bug. List-patterns is in active development, so we're tracking work remaining in the test plan.
Also, I have a PR queued that fixes that for list-patterns already (it's waiting on another PR to merge first) ;-)
https://github.com/dotnet/roslyn/pull/53850/files#diff-31303c421816ee9001acbb872ec31e6ef3d701cd04eba6f2a41d02e49d70ae21